Library/SPIClient.m

```diff
@@ -1170,6 +1171,26 @@ - (void)autoResolveEftposAddress {
     }];
 }
 
+#pragma Static Methods
+
+/**
```

Comment:
You don't need to repeat the comment block in the implementation, header file one is sufficient.